src/themes/money/theme.json

```diff
@@ -392,7 +392,10 @@
         },
         "centered": {
           "display": "inline-block",
-          "align-items": "center"
+          "align-items": "center",
+          "text": {
+            "color": "white"
```

Comment:
this isn't always on a dark background so it has to be a separate variant

Reply:
I just updated, is that what you're looking for?

Reply:
sure if this works it will be much better :D